packages/uswds-core/src/js/utils/test/keymap.spec.js

```diff
@@ -0,0 +1,65 @@
+const sinon = require("sinon");
+const assert = require("assert");
+const keymap = require("../keymap");
+
+describe("keymap", () => {
+  it("does not call handler if keypress is not for the mapped key", () => {
+    const onTab = sinon.stub();
+    const handler = keymap({ Tab: onTab });
+    document.body.addEventListener("keydown", handler);
+    const event = new KeyboardEvent("keydown", { key: "Escape" });
+    document.body.dispatchEvent(event);
+
+    assert(onTab.notCalled);
+  });
+
+  it("calls handler if combo does not include modifier and neither does keypress", () => {
```

Comment:
**polish:** This test description is a bit confusing. 

I read this as: "It calls the event handler if the combination of keys does not include a modifier or the pressed key". Which can't be the case because we are pressing the `tab` key in this test.

If I'm understanding correctly, maybe something like:

```suggestion
  it("calls event if keypress matches keymap handler", () => {
```

Reply:
Yeah, the purpose of this is to verify the handler being called with an unqualified (no modifier) keypress, i.e. only if both the keypress event has no modifier and the expected key combination has no modifier.

Maybe something like ... ?

```suggestion
  it("calls handler if keypress matches keymap handler with no modifiers", () => {
```

Reply:
**Suggestion:** Match new language used in other tests for consistency:

```suggestion
  it("calls handler if when no modifiers are expected or pressed", () => {
```